balance_check/providers/gamestop.py

```diff
@@ -0,0 +1,111 @@
+            sys.exit(1)
+
+        page_html = BeautifulSoup(resp.content, features="html.parser")
+        form = page_html.find("form")
+        if not form:
+            logger.critical("Unable to find balance check form")
+            sys.exit(1)
+
+        endpoint = "{}{}".format(self.website_url, form['action'])
+
+        # These fields are present on GS balance check page, does not work without including them
+        fields['__EVENTTARGET'] = ''
+        fields['__EVENTARGUMENT'] = ''
+        fields['__LASTFOCUS'] = ''
+        fields['__VIEWSTATE'] = page_html.find("input",id='__VIEWSTATE')['value']
+        fields['__VIEWSTATEGENERATOR'] = page_html.find("input",id='__VIEWSTATEGENERATOR')['value']
+
+        recaptcha_field = page_html.find("div", class_="g-recaptcha")
+        if not recaptcha_field:
+            logger.critical("Unable to find reCAPTCHA")
+            sys.exit(1)
+
+        site_key = recaptcha_field["data-sitekey"]
+
+        logger.info("Solving reCAPTCHA (~30s)")
+
+        captcha_resp = captcha_solver.solve_recaptcha(self.website_url, site_key)
+        if captcha_resp["errorId"] != 0:
+            logger.critical("Unable to solve reCAPTCHA ({})".format(captcha_resp["errorDescription"]))
+            sys.exit(1)
+
+        fields["g-recaptcha-response"] = captcha_resp["solution"]["gRecaptchaResponse"]
+
+        logger.info("Fetching card balance")
+
+        session.headers.update({
+            "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8",
+            "Accept-Encoding": "gzip, deflate, br",
+            "Accept-Language": "en-US,en;q=0.9",
+            "Cache-Control": "max-age=0",
+            "Connection": "keep-alive",
+            "Content-Type": "application/x-www-form-urlencoded",
+            "Pragma": "no-cache",
+            "Referer": "https://www.gamestop.com/profiles/valuelookup.aspx",
+            "X-Requested-With": "XMLHttpRequest"
+        })
+
+        form_resp = session.post(endpoint, data=fields)
+        if form_resp.status_code != 200:
+            logger.critical("Failed to retrieve card balance (status code {})".format(form_resp.status_code))
+            sys.exit(1)
+
+        balance_html = BeautifulSoup(form_resp.content, features="html.parser")
+
+        try:
+            avail_balance = balance_html.find("span",class_='balancePrice').text
+        except:
+            # GS prunes old depleted cards from its system and throws an 'invalid' error
+            # Set these as -1 to notate they gave invalid error but are likely actually zero balance
+            if balance_html.find(text='The Gift Card number entered is invalid.'):
+                avail_balance = '-1' 
+            else:
+                print('ERROR finding balance on page')
+                raise
+
+
+        logger.info("Success! Card balance: {}".format(avail_balance))
+
+        return ({
+            "available": avail_balance
```

Comment:
I'd suggest just using `balance` if not differentiating between initial/available balances.
```suggestion
            "balance": avail_balance
```